docs/docs/Deployment/deployment-enterprise-database.mdx

```diff
@@ -0,0 +1,271 @@
+
+2. Obtain the connection string in the format `postgresql://user:password@host:port/dbname`, such as`postgresql://langflow:securepassword@postgres:5432/langflow`.
+
+   For High Availability setups, use a virtual IP or proxy hostname. For more information, see [High Availability for PostgreSQL](#high-availability-ha-for-postgresql).
+
+3. Configure Langflow with the `.env` or `docker-compose.yml` files.
+
+    <Tabs groupId="environment">
+      <TabItem value=".env" label=".env" default>
+
+        i. Create a `.env` file in the Langflow directory: `touch .env`.
+
+        ii. Add the connection string to the `.env` file: `LANGFLOW_DATABASE_URL="postgresql://langflow:securepassword@postgres:5432/langflow"`.
+
+        Reference the `.env.example` file from the Langflow GitHub Repository for additional settings.
+
+      </TabItem>
+      <TabItem value="docker-compose.yml" label="docker-compose.yml">
+
+        Use or modify the example `docker-compose.yml` from the Langflow GitHub Repository.
+
+        ```yaml
+        version: '3'
+        services:
+          langflow:
+            image: langflowai/langflow:latest
+            ports:
+              - "7860:7860"
+            environment:
+              - LANGFLOW_DATABASE_URL=postgresql://langflow:langflow@postgres:5432/langflow
+          postgres:
+            image: postgres:16
+            ports:
+              - "5432:5432"
+            environment:
+              - POSTGRES_USER=langflow
+              - POSTGRES_PASSWORD=langflow
+              - POSTGRES_DB=langflow
+            volumes:
+              - langflow-postgres:/var/lib/postgresql/data
+        volumes:
+          - langflow-postgres:
+        ```
+
+      </TabItem>
+    </Tabs>
+
+
+4. Start Langflow with the value for the connection string.
+
+    <Tabs groupId="environment">
+      <TabItem value=".env" label=".env" default>
+
+      `uv run langflow run --env-file .env`
+
+      </TabItem>
+      <TabItem value="docker-compose.yml" label="docker-compose.yml">
+
+      Navigate to the directory containing `docker-compose.yml` and run `docker-compose up`.
+
+      </TabItem>
+    </Tabs>
+
+5. Optional: Run migrations.
+   Langflow uses SQLAlchemy and Alembic to manage its database schema. When you first connect to PostgreSQL, Langflow will automatically run migrations to create the necessary tables. To manually run migrations, use `langflow migration --no-test`. To preview changes before applying them, use `langflow migration --test`.
+
+5. To verify the configuration, create a test flow using the UI or API and inspect the PostgreSQL database to confirm new tables and activity. You can query the database container with `docker exec -it <postgres-container> psql -U langflow -d langflow` or use `SELECT * FROM pg_stat_activity WHERE datname = 'langflow';` to verify connectivity.
```

Comment:
~~~suggestion
5. To verify the configuration, create any flow using the Langflow UI or API, and then query your database to confirm new tables and activity. The content of the flow doesn't matter; you only need to confirm that the flow is stored in your PostgreSQL database.
    
    * Query the database container:
    
        ```
        docker exec -it <postgres-container> psql -U langflow -d langflow
        ```
    
    * Use SQL:
    
        ```
        SELECT * FROM pg_stat_activity WHERE datname = 'langflow';
        ```
~~~